draft-ietf-quic-multipath.md

```diff
@@ -543,22 +516,21 @@ MUST send a PATH_ABANDON frame to retire the Path ID.
 
 ### Allocating, Consuming and Retiring Connection IDs {#consume-retire-cid}
 
-Each endpoints pre-allocate a Path Identifier for each new Connection ID.
+Each endpoints pre-allocate a Path Identifier for each new connection ID.
```

Comment:
That's incorrect. Multiple CIDs are associated with the same path ID.

Reply:
This is correct because it doesn't say that it can be the same path ID. However, I agree it's confusing.

Reply:
Not fully sure what to do here. Maybe just remove the sentence entirely?
```suggestion
```

Reply:
I'd suggest change the sentence into: 
Each endpoints _assign_ a Path Identifier for each new connection ID.

Reply:
I'd argue it's the wrong way around anyway. **First** you create the path ID, **then** you issue CIDs for this path.

Reply:
I don't think that is a useful sentence. I think we can just remove it. Or what do you think is missing if we remove it, @Yanmei-Liu ?

Reply:
This sentence is certainly necessary, we should clarify the relationship between CID and PATH ID at the very beginning of this section.